data/macros.meson

```diff
@@ -1,6 +1,7 @@
 %__meson %{_bindir}/meson
 %__meson_wrap_mode nodownload
 %__meson_auto_features enabled
+%__meson_verbose 1
```

Comment:
I wanted to ask whether making this `0|1` wouldn't be better, but then I remembered that rpm would require awkward syntax of `--define '__meson_verbose 0'`. `--undefine` is more user friendly.